_data/sponsors.yml

```diff
@@ -30,43 +18,60 @@
   image: firsttech.jpg
   website: https://www.firsttechfed.com/
   tier: Gold
-- name: City of Eugene
-  image: eugene.png
-  website: https://www.eugene-or.gov/
-  tier: Silver
-- name: John Deere
-  image: johndeere.png
-  website: https://www.deere.com/
-  tier: Diamond
 - name: K Ridley Tech
   image: kridley.jpg
   website: https://www.kridleytech.com/
   tier: Gold
-- name: Marathon
-  image: marathon.png
-  website: https://www.marathonpetroleum.com/
-  tier: Silver
 - name: The Oregon Department of Education
   image: ode.png
   website: https://www.oregon.gov/ode/Pages/default.aspx
   tier: Platinum
-- name: OMRON
-  image: omron.svg
-  website: https://www.omron.com/
-  tier: Silver
 - name: South Eugene High School
   image: sehs.png
   website: https://sehs.4j.lane.edu/
   tier: Silver
 - name: Solidworks
   image: solidworks.png
   website: https://www.solidworks.com/
-  tier: In-Kind
-- name: City of Springfield
-  image: springfield.jpg
-  website: https://www.springfield-or.gov/
-  tier: Silver
-- name: XS Media
-  image: xsmedia.png
-  website: https://xsmedia.com/
   tier: Platinum
+- name: Olive Garden
+  image: 1200px-Olive_Garden_Logo.svg.png
+  website: https://www.olivegarden.com/
+  tier: In-Kind
+- name: MitoSciences
+  image: mitosciences.png
+  website: https://www.abcam.com/index.html?pageconfig=resource&rid=13917
+  tier: Bronze
+- name: OCCU
+  image: OCCU_Logo_RGB_Color_2BDEE07F-1600-49C6-96E85C6D2E6DE6E3_90ff9118-1fa6-4d2b-847ec746a87bef0b.png
+  website: https://www.myoccu.org/
+  tier: Bronze
+- name: Tripwire
+  image: Tripwire logo.png
+  website: https://www.tripwire.com/
+  tier: Gold
+- name: Boutique Academia
+  image: BoutiqueAcademiaShopifyBW_193d0c5f-0103-43c8-9151-d49c4e8798c0_410x.png
+  website: https://www.boutiqueacademia.com/
+  tier: In-Kind
+- name: Office Max
+  image: Office-Depot-logo-header-817x213.jpg
+  website: https://www.officedepot.com/
+  tier: In-Kind
+- name: SERT friends and family
+  image: 
+  website: 
+  tier: In-Kind
```

Comment:
This just shows up as:

![Screenshot from 2020-04-16 16-15-25](https://user-images.githubusercontent.com/10191084/79515428-75ead880-7ffd-11ea-8961-4918210ea4a5.png)

We can either make modifications to the sponsor code to allow text as a backup, or create an image with the SERT logo and "Friends & Family" (or just remove this entirely until someone has time to work on that modification).

Reply:
sense we need it pretty soon, I think we can just crop it our of the code, and I'll create the logo.